content/en/docs/tasks/debug-application-cluster/debug-running-pod.md

```diff
@@ -0,0 +1,189 @@
+---
+reviewers:
+- verb
+- soltysh
+title: Debug Running Pods
+content_template: templates/task
+---
+
+{{% capture overview %}}
+
+This page explains how to debug Pods running (or crashing) on a Node.
+
+{{% /capture %}}
+
+{{% capture prerequisites %}}
+
+* Your {{< glossary_tooltip text="Pod" term_id="pod" >}} should already be
+  scheduled and running. If your Pod is not yet running, start with [Troubleshoot
+  Applications](/docs/tasks/debug-application-cluster/debug-application/).
+* You should be familiar with {{< glossary_tooltip text="Ephemeral Containers"
+  term_id="ephemeral-container" >}}.
+
```

Comment:
Does the reader need to know what node the Pod is scheduled onto?

Reply:
Only for the very last item, to get a shell on that node. All of the `kubectl` commands are independent of node.

Reply:
(done)